devolutions-agent/src/config.rs

```diff
@@ -160,6 +122,14 @@ pub fn load_conf_file_or_generate_new() -> anyhow::Result<dto::ConfFile> {
 pub mod dto {
     use super::*;
 
+    #[derive(PartialEq, Eq, Debug, Default, Clone, Serialize, Deserialize)]
+    #[serde(rename_all = "PascalCase")]
+    pub struct UpdaterConf {
+        /// Enable updater module (enabled by default)
+        #[serde(default)]
+        pub disable: bool,
```

Comment:
suggestion: I think it would be better the other way around. You override the default value to "true" if that’s what you want.
```suggestion
        /// Enable updater module (enabled by default)
        #[serde(default = "true")]
        pub enabled: bool,
```